### PR TITLE
Atomization of the components

### PR DESCRIPTION
**Pull request by balintcristian**
De-structured components to atomic levels and moved them in corresponding directories.
Import arranged for readability.

### CODE REVIEW

File: src/components/atoms/FooterIcon/FooterIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import Image from "next/image";
```
Comment (robibalint) [inline src/components/atoms/FooterIcon/FooterIcon.tsx]: Let's make this a general Icon, not just for Footer

File: src/components/molecules/FooterIconLink/FooterIconLink.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import FooterIcon from "@/components/atoms/FooterIcon/FooterIcon";
+
```
Comment (robibalint) [inline src/components/molecules/FooterIconLink/FooterIconLink.tsx]: Same. Let's make this IconLink

File: src/components/molecules/GalleryCard/GalleryCard.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+const GalleryImage = ({ src, description }: GalleryImageProps) => {
+  return (
+    // <div>
+    <Image
+      src={src}
+      alt="nature"
+      width={0}
+      height={0}
+      sizes="100vw"
+      style={{ width: "100%", height: "100%" }}
```
Comment (robibalint) [inline src/components/molecules/GalleryCard/GalleryCard.tsx]: Don't use inline styling. Let's stick to scss please. Why did you set `width` & `height` to 0 above?

Comment (balintcristian) [reply src/components/molecules/GalleryCard/GalleryCard.tsx]: Cause i don't have de widths and heights and next js requires them for their image component. So i thought i'd be better to do it that way for now, until i find a better solution. Thanks for reminding me.

File: src/components/molecules/FooterIconLink/FooterIconLink.tsx (hunk 2 above)
Comment (balintcristian) [reply src/components/molecules/FooterIconLink/FooterIconLink.tsx]: On it, on the icon changes

**Comment by robibalint** [comment]
Also, shouldn't each of these components have a related scss file?

File: src/components/molecules/GalleryCard/GalleryCard.tsx (hunk 3 above)
Comment (robibalint) [reply src/components/molecules/GalleryCard/GalleryCard.tsx]: Then maybe stick to a plain `` tag?

Comment (balintcristian) [reply src/components/molecules/GalleryCard/GalleryCard.tsx]: Your call, it's a fair point after all. My point was that next js recommends those for better loading times. I can get the the inline style out though and make it a simple image i guess.

**Comment by balintcristian** [comment]
Working on the scss files as we speak

**Comment by balintcristian** [comment]
Renamed GalleryCard to GalleryImage, the whole code was written as GalleryImage and removed the inline stylings. Though, i could go back to gallery card.

**Review by robibalint — APPROVED** [review]

**Pull request merged**